src/components/Reports/LostTime/EditHistoryModal.jsx

```diff
@@ -88,7 +89,7 @@ const EditHistoryModal = props => {
     if (props.entryType == 'project') {
       return (
         <FormGroup>
-          <Label>Project Name</Label>
+          <Label className={darkMode ? 'text-light' : ''}>Project Name</Label>
```

Comment:
Same recommendation as above. If we are repeating lines of code 2 or more times it's better to have it stored in a variable.